Resources/Private/JavaScript/Components/Dialog/style.css

```diff
@@ -16,8 +16,12 @@
 }
 .dialog__contents {
     position: relative;
-    width: 80vw;
-    min-height: 300px;
+    width: 50vw;
```

Comment:
@aertmann posted this in slack, just as a reminder :) :+1:
https://github.com/neos/neos-development-collection/commit/dd81235f0bb9cb455867ff182e6db42617e138ab